tools/isobuild/import-scanner.js

```diff
@@ -646,7 +639,7 @@ export default class ImportScanner {
       depFile.installPath = installPath;
       depFile.servePath = installPath;
       depFile.lazy = true;
-      depFile.imported = dynamic ? "dynamic" : true;
+      depFile.imported = false;
```

Comment:
This is only temporary, since we're going to call `this._scanFile(depFile, dynamic)` below.